spacewalk-utils support (#223)
uyuni-project/uyuni-docs: modules/client-configuration/pages/supported-features.adoc

| {ubuntu} 18.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
| {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
|===

== Supported Tools Packages

> The ``spacewalk-utils`` package is an additional package for {productname} that can provide additional services and features.

Reviewer: ```suggestion
The ``spacewalk-utils`` and ``spacewalk-utils-extras`` packages can provide additional services and features.
```

> 

Reviewer: I think we do not need the historical and version details here.  This info is something for the release notes only.

Author: I added that because I was fairly certain we used to have some warnings about "`spacewalk-utils` is not supported" but then I couldn't find it. Happy to take it out, though.

In {productname}{nbsp}4.0 and earlier, the ``spacewalk-utils`` package contained a mixture of supported and unsupported tools.
In {productname}{nbsp}4.1 and later, the package has been split into two, with different levels of support.

The ``spacewalk-utils`` package in {productname}{nbsp}4.1 and later is fully supported by {suse}, and contains these tools:

[[spacewalk-utils]]
[cols="1,1,1", options="header"]

> .Spacewalk-utils Tools

Reviewer: ```suggestion
.Spacewalk Utilities
```
Or "Package spacewalk-utils", but better not a mixture of everything ;)
